Controller Spawner couldn't find the expected controller_manager ROS interface.

<!-- 品質向上にご協力ありがとうございます -->
<!-- テンプレートを参考に文章を作成してください -->

# 不具合の概要
<!-- 不具合の概要をここに書きます -->

`roslaunch crane_x7_gazebo crane_x7_with_table.launch` で以下のような警告が出る

```
[WARN] [1648875337.082548, 27.536000]: Controller Spawner couldn't find the expected controller_manager ROS interface.
[controller_gazebo-3] process has finished cleanly
log file: /home/daisuke/.ros/log/0fef90ac-b241-11ec-b3b9-a85e45a7a8b0/controller_gazebo-3*.log
```


# 実行環境
<!-- 不具合発生時の実行環境をここに書きます -->

- OS: <!-- 例：Ubuntu 18.04、Windows 10 -->Ubuntu 18.04 + ROS Melodic
- ソフトのバージョン: <!-- 例：○月○日のmasterの最新、コミットabcd123 -->[5578110](https://github.com/rt-net/crane_x7_ros/commit/5578110f20ce72e210ecef0e05ebbec1237f9d83)

# 再現方法
<!-- 不具合の再現方法をここに書きます -->

1. `roslaunch crane_x7_gazebo crane_x7_with_table.launch` を実行する
2. MoveItが起動してから10秒〜1分ほど待つ
3. `Controller Spawner couldn't find the expected controller_manager ROS interface.`と警告が表示される

# 期待する動作

警告なしでgazebo_ros_controlがロードされる（ログを見るとgazebo_ros_controlのロードはされているもよう）

# ログ・画像
<!-- 不具合発生時のログや画像を貼ります -->
<!-- 画像はコピー&ペーストや、ドラッグ&ドロップで貼り付けられます -->

```
$ roslaunch crane_x7_gazebo crane_x7_with_table.launch
... logging to /home/daisuke/.ros/log/0fef90ac-b241-11ec-b3b9-a85e45a7a8b0/roslaunch-i59600k-8671.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

xacro: in-order processing became default in ROS Melodic. You can drop the option.
started roslaunch server http://192.168.100.2:36501/

SUMMARY
========

PARAMETERS
 * /crane_x7/arm_controller/constraints/goal_time: 0.0
 * /crane_x7/arm_controller/constraints/stopped_velocity_tolerance: 1.0
 * /crane_x7/arm_controller/joints: ['crane_x7_should...
 * /crane_x7/arm_controller/publish_rate: 100
 * /crane_x7/arm_controller/type: position_controll...
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_gripper_finger_a_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_gripper_finger_a_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_gripper_finger_a_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_gripper_finger_b_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_gripper_finger_b_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_gripper_finger_b_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_lower_arm_fixed_part_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_lower_arm_fixed_part_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_lower_arm_fixed_part_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_lower_arm_revolute_part_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_lower_arm_revolute_part_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_lower_arm_revolute_part_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_shoulder_fixed_part_pan_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_shoulder_fixed_part_pan_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_shoulder_fixed_part_pan_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_shoulder_revolute_part_tilt_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_shoulder_revolute_part_tilt_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_shoulder_revolute_part_tilt_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_upper_arm_revolute_part_rotate_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_upper_arm_revolute_part_rotate_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_upper_arm_revolute_part_rotate_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_upper_arm_revolute_part_twist_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_upper_arm_revolute_part_twist_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_upper_arm_revolute_part_twist_joint/p: 100.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_wrist_joint/d: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_wrist_joint/i: 0.0
 * /crane_x7/gazebo_ros_control/pid_gains/crane_x7_wrist_joint/p: 100.0
 * /crane_x7/gripper_controller/action_monitor_rate: 50
 * /crane_x7/gripper_controller/action_ns: gripper_cmd
 * /crane_x7/gripper_controller/gains/crane_x7_gripper_finger_a_joint/d: 0.0
 * /crane_x7/gripper_controller/gains/crane_x7_gripper_finger_a_joint/i: 0.0
 * /crane_x7/gripper_controller/gains/crane_x7_gripper_finger_a_joint/p: 100.0
 * /crane_x7/gripper_controller/goal_tolerance: 0.2
 * /crane_x7/gripper_controller/joint: crane_x7_gripper_...
 * /crane_x7/gripper_controller/publish_rate: 100
 * /crane_x7/gripper_controller/stall_timeout: 0.3
 * /crane_x7/gripper_controller/stall_velocity_threshold: 0.01
 * /crane_x7/gripper_controller/state_publish_rate: 100
 * /crane_x7/gripper_controller/type: position_controll...
 * /crane_x7/joint_state_controller/publish_rate: 100
 * /crane_x7/joint_state_controller/type: joint_state_contr...
 * /gazebo/enable_ros_network: True
 * /move_group/allow_trajectory_execution: True
 * /move_group/arm/planner_configs: ['SBLkConfigDefau...
 * /move_group/controller_list: [{'default': True...
 * /move_group/gripper/longest_valid_segment_fraction: 0.005
 * /move_group/gripper/planner_configs: ['SBLkConfigDefau...
 * /move_group/gripper/projection_evaluator: joints(crane_x7_g...
 * /move_group/jiggle_fraction: 0.05
 * /move_group/max_range: 5.0
 * /move_group/max_safe_path_cost: 1
 * /move_group/moveit_controller_manager: moveit_simple_con...
 * /move_group/moveit_manage_controllers: True
 * /move_group/octomap_resolution: 0.025
 * /move_group/planner_configs/BFMTkConfigDefault/balanced: 0
 * /move_group/planner_configs/BFMTkConfigDefault/cache_cc: 1
 * /move_group/planner_configs/BFMTkConfigDefault/extended_fmt: 1
 * /move_group/planner_configs/BFMTkConfigDefault/heuristics: 1
 * /move_group/planner_configs/BFMTkConfigDefault/nearest_k: 1
 * /move_group/planner_configs/BFMTkConfigDefault/num_samples: 1000
 * /move_group/planner_configs/BFMTkConfigDefault/optimality: 1
 * /move_group/planner_configs/BFMTkConfigDefault/radius_multiplier: 1.0
 * /move_group/planner_configs/BFMTkConfigDefault/type: geometric::BFMT
 * /move_group/planner_configs/BKPIECEkConfigDefault/border_fraction: 0.9
 * /move_group/planner_configs/BKPIECEkConfigDefault/failed_expansion_score_factor: 0.5
 * /move_group/planner_configs/BKPIECEkConfigDefault/min_valid_path_fraction: 0.5
 * /move_group/planner_configs/BKPIECEkConfigDefault/range: 0.0
 * /move_group/planner_configs/BKPIECEkConfigDefault/type: geometric::BKPIECE
 * /move_group/planner_configs/BiESTkConfigDefault/range: 0.0
 * /move_group/planner_configs/BiESTkConfigDefault/type: geometric::BiEST
 * /move_group/planner_configs/BiTRRTkConfigDefault/cost_threshold: 1e300
 * /move_group/planner_configs/BiTRRTkConfigDefault/frountier_node_ratio: 0.1
 * /move_group/planner_configs/BiTRRTkConfigDefault/frountier_threshold: 0.0
 * /move_group/planner_configs/BiTRRTkConfigDefault/init_temperature: 100
 * /move_group/planner_configs/BiTRRTkConfigDefault/range: 0.0
 * /move_group/planner_configs/BiTRRTkConfigDefault/temp_change_factor: 0.1
 * /move_group/planner_configs/BiTRRTkConfigDefault/type: geometric::BiTRRT
 * /move_group/planner_configs/ESTkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/ESTkConfigDefault/range: 0.0
 * /move_group/planner_configs/ESTkConfigDefault/type: geometric::EST
 * /move_group/planner_configs/FMTkConfigDefault/cache_cc: 1
 * /move_group/planner_configs/FMTkConfigDefault/extended_fmt: 1
 * /move_group/planner_configs/FMTkConfigDefault/heuristics: 0
 * /move_group/planner_configs/FMTkConfigDefault/nearest_k: 1
 * /move_group/planner_configs/FMTkConfigDefault/num_samples: 1000
 * /move_group/planner_configs/FMTkConfigDefault/radius_multiplier: 1.1
 * /move_group/planner_configs/FMTkConfigDefault/type: geometric::FMT
 * /move_group/planner_configs/KPIECEkConfigDefault/border_fraction: 0.9
 * /move_group/planner_configs/KPIECEkConfigDefault/failed_expansion_score_factor: 0.5
 * /move_group/planner_configs/KPIECEkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/KPIECEkConfigDefault/min_valid_path_fraction: 0.5
 * /move_group/planner_configs/KPIECEkConfigDefault/range: 0.0
 * /move_group/planner_configs/KPIECEkConfigDefault/type: geometric::KPIECE
 * /move_group/planner_configs/LBKPIECEkConfigDefault/border_fraction: 0.9
 * /move_group/planner_configs/LBKPIECEkConfigDefault/min_valid_path_fraction: 0.5
 * /move_group/planner_configs/LBKPIECEkConfigDefault/range: 0.0
 * /move_group/planner_configs/LBKPIECEkConfigDefault/type: geometric::LBKPIECE
 * /move_group/planner_configs/LBTRRTkConfigDefault/epsilon: 0.4
 * /move_group/planner_configs/LBTRRTkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/LBTRRTkConfigDefault/range: 0.0
 * /move_group/planner_configs/LBTRRTkConfigDefault/type: geometric::LBTRRT
 * /move_group/planner_configs/LazyPRMkConfigDefault/range: 0.0
 * /move_group/planner_configs/LazyPRMkConfigDefault/type: geometric::LazyPRM
 * /move_group/planner_configs/LazyPRMstarkConfigDefault/type: geometric::LazyPR...
 * /move_group/planner_configs/PDSTkConfigDefault/type: geometric::PDST
 * /move_group/planner_configs/PRMkConfigDefault/max_nearest_neighbors: 10
 * /move_group/planner_configs/PRMkConfigDefault/type: geometric::PRM
 * /move_group/planner_configs/PRMstarkConfigDefault/type: geometric::PRMstar
 * /move_group/planner_configs/ProjESTkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/ProjESTkConfigDefault/range: 0.0
 * /move_group/planner_configs/ProjESTkConfigDefault/type: geometric::ProjEST
 * /move_group/planner_configs/RRTConnectkConfigDefault/range: 0.0
 * /move_group/planner_configs/RRTConnectkConfigDefault/type: geometric::RRTCon...
 * /move_group/planner_configs/RRTkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/RRTkConfigDefault/range: 0.0
 * /move_group/planner_configs/RRTkConfigDefault/type: geometric::RRT
 * /move_group/planner_configs/RRTstarkConfigDefault/delay_collision_checking: 1
 * /move_group/planner_configs/RRTstarkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/RRTstarkConfigDefault/range: 0.0
 * /move_group/planner_configs/RRTstarkConfigDefault/type: geometric::RRTstar
 * /move_group/planner_configs/SBLkConfigDefault/range: 0.0
 * /move_group/planner_configs/SBLkConfigDefault/type: geometric::SBL
 * /move_group/planner_configs/SPARSkConfigDefault/dense_delta_fraction: 0.001
 * /move_group/planner_configs/SPARSkConfigDefault/max_failures: 1000
 * /move_group/planner_configs/SPARSkConfigDefault/sparse_delta_fraction: 0.25
 * /move_group/planner_configs/SPARSkConfigDefault/stretch_factor: 3.0
 * /move_group/planner_configs/SPARSkConfigDefault/type: geometric::SPARS
 * /move_group/planner_configs/SPARStwokConfigDefault/dense_delta_fraction: 0.001
 * /move_group/planner_configs/SPARStwokConfigDefault/max_failures: 5000
 * /move_group/planner_configs/SPARStwokConfigDefault/sparse_delta_fraction: 0.25
 * /move_group/planner_configs/SPARStwokConfigDefault/stretch_factor: 3.0
 * /move_group/planner_configs/SPARStwokConfigDefault/type: geometric::SPARStwo
 * /move_group/planner_configs/STRIDEkConfigDefault/degree: 16
 * /move_group/planner_configs/STRIDEkConfigDefault/estimated_dimension: 0.0
 * /move_group/planner_configs/STRIDEkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/STRIDEkConfigDefault/max_degree: 18
 * /move_group/planner_configs/STRIDEkConfigDefault/max_pts_per_leaf: 6
 * /move_group/planner_configs/STRIDEkConfigDefault/min_degree: 12
 * /move_group/planner_configs/STRIDEkConfigDefault/min_valid_path_fraction: 0.2
 * /move_group/planner_configs/STRIDEkConfigDefault/range: 0.0
 * /move_group/planner_configs/STRIDEkConfigDefault/type: geometric::STRIDE
 * /move_group/planner_configs/STRIDEkConfigDefault/use_projected_distance: 0
 * /move_group/planner_configs/TRRTkConfigDefault/frountierNodeRatio: 0.1
 * /move_group/planner_configs/TRRTkConfigDefault/frountier_threshold: 0.0
 * /move_group/planner_configs/TRRTkConfigDefault/goal_bias: 0.05
 * /move_group/planner_configs/TRRTkConfigDefault/init_temperature: 10e-6
 * /move_group/planner_configs/TRRTkConfigDefault/k_constant: 0.0
 * /move_group/planner_configs/TRRTkConfigDefault/max_states_failed: 10
 * /move_group/planner_configs/TRRTkConfigDefault/min_temperature: 10e-10
 * /move_group/planner_configs/TRRTkConfigDefault/range: 0.0
 * /move_group/planner_configs/TRRTkConfigDefault/temp_change_factor: 2.0
 * /move_group/planner_configs/TRRTkConfigDefault/type: geometric::TRRT
 * /move_group/planning_plugin: ompl_interface/OM...
 * /move_group/planning_scene_monitor/publish_geometry_updates: True
 * /move_group/planning_scene_monitor/publish_planning_scene: True
 * /move_group/planning_scene_monitor/publish_state_updates: True
 * /move_group/planning_scene_monitor/publish_transforms_updates: True
 * /move_group/request_adapters: default_planner_r...
 * /move_group/start_state_max_bounds_error: 0.1
 * /move_group/trajectory_execution/allowed_execution_duration_scaling: 0.6
 * /move_group/trajectory_execution/allowed_goal_duration_margin: 0.5
 * /move_group/trajectory_execution/allowed_start_tolerance: 0.1
 * /robot_description: <?xml version="1....
 * /robot_description_kinematics/arm/kinematics_solver: kdl_kinematics_pl...
 * /robot_description_kinematics/arm/kinematics_solver_search_resolution: 0.005
 * /robot_description_kinematics/arm/kinematics_solver_timeout: 0.005
 * /robot_description_planning/default_acceleration_scaling_factor: 1.0
 * /robot_description_planning/default_velocity_scaling_factor: 1.0
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_a_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_a_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_a_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_a_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_b_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_b_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_b_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_gripper_finger_b_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_fixed_part_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_fixed_part_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_fixed_part_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_fixed_part_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_revolute_part_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_revolute_part_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_revolute_part_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_lower_arm_revolute_part_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_shoulder_fixed_part_pan_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_shoulder_fixed_part_pan_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_shoulder_fixed_part_pan_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_shoulder_fixed_part_pan_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_shoulder_revolute_part_tilt_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_shoulder_revolute_part_tilt_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_shoulder_revolute_part_tilt_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_shoulder_revolute_part_tilt_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_rotate_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_rotate_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_rotate_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_rotate_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_twist_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_twist_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_twist_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_upper_arm_revolute_part_twist_joint/max_velocity: 4.81710873
 * /robot_description_planning/joint_limits/crane_x7_wrist_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/crane_x7_wrist_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/crane_x7_wrist_joint/max_acceleration: 4.4
 * /robot_description_planning/joint_limits/crane_x7_wrist_joint/max_velocity: 4.81710873
 * /robot_description_semantic: <?xml version="1....
 * /rosdistro: melodic
 * /rosversion: 1.14.12
 * /rviz_i59600k_8671_2854060825733414977/arm/kinematics_solver: kdl_kinematics_pl...
 * /rviz_i59600k_8671_2854060825733414977/arm/kinematics_solver_search_resolution: 0.005
 * /rviz_i59600k_8671_2854060825733414977/arm/kinematics_solver_timeout: 0.005
 * /source_list: ['/crane_x7/joint...
 * /use_sim_time: True

NODES
  /
    controller_gazebo (controller_manager/spawner)
    gazebo (gazebo_ros/gzserver)
    gazebo_gui (gazebo_ros/gzclient)
    joint_state_publisher (joint_state_publisher/joint_state_publisher)
    move_group (moveit_ros_move_group/move_group)
    robot_state_publisher (robot_state_publisher/robot_state_publisher)
    rviz_i59600k_8671_2854060825733414977 (rviz/rviz)
    urdf_spawner (gazebo_ros/spawn_model)
  /crane_x7/
    controller_manager (controller_manager/spawner)

auto-starting new master
process[master]: started with pid [8686]
ROS_MASTER_URI=http://192.168.100.2:11311

setting /run_id to 0fef90ac-b241-11ec-b3b9-a85e45a7a8b0
process[rosout-1]: started with pid [8699]
started core service [/rosout]
process[crane_x7/controller_manager-2]: started with pid [8707]
process[controller_gazebo-3]: started with pid [8708]
process[gazebo-4]: started with pid [8709]
process[gazebo_gui-5]: started with pid [8714]
process[urdf_spawner-6]: started with pid [8718]
process[joint_state_publisher-7]: started with pid [8720]
process[robot_state_publisher-8]: started with pid [8721]
process[move_group-9]: started with pid [8722]
process[rviz_i59600k_8671_2854060825733414977-10]: started with pid [8732]
[ INFO] [1648875306.725756302]: Loading robot model 'crane_x7'...
[ INFO] [1648875306.726363420]: No root/virtual joint specified in SRDF. Assuming fixed joint
[ INFO] [1648875306.803483913]: rviz version 1.13.23
[ INFO] [1648875306.803518116]: compiled against Qt version 5.9.5
[ INFO] [1648875306.803525424]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [1648875306.824096236]: Forcing OpenGl version 0.
[ INFO] [1648875306.850807708]: Publishing maintained planning scene on 'monitored_planning_scene'
[ INFO] [1648875306.852096451]: MoveGroup debug mode is ON
Starting planning scene monitors...
[ INFO] [1648875306.852134868]: Starting planning scene monitor
[ INFO] [1648875306.853925899]: Listening to '/planning_scene'
[ INFO] [1648875306.853973354]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[ INFO] [1648875306.855056953]: Listening to '/collision_object'
[ INFO] [1648875306.856421569]: Listening to '/planning_scene_world' for planning scene world geometry
[ INFO] [1648875306.856995775]: No 3D sensor plugin(s) defined for octomap updates
[ INFO] [1648875306.867992481]: Listening to '/attached_collision_object' for attached collision objects
Planning scene monitors started.
[ INFO] [1648875306.889347968]: Initializing OMPL interface using ROS parameters
[ INFO] [1648875306.914179271]: Using planning interface 'OMPL'
[ INFO] [1648875306.917147051]: Param 'default_workspace_bounds' was not set. Using default value: 10
[ INFO] [1648875306.917869639]: Param 'start_state_max_bounds_error' was set to 0.1
[INFO] [1648875306.918108, 0.000000]: Controller Spawner: Waiting for service controller_manager/load_controller
[ INFO] [1648875306.918438729]: Param 'start_state_max_dt' was not set. Using default value: 0.5
[ INFO] [1648875306.919059278]: Param 'start_state_max_dt' was not set. Using default value: 0.5
[ INFO] [1648875306.919357783]: Param 'jiggle_fraction' was set to 0.05
[ INFO] [1648875306.919621199]: Param 'max_sampling_attempts' was not set. Using default value: 100
[ INFO] [1648875306.919675064]: Using planning request adapter 'Add Time Parameterization'
[ INFO] [1648875306.919697624]: Using planning request adapter 'Fix Workspace Bounds'
[ INFO] [1648875306.919721592]: Using planning request adapter 'Fix Start State Bounds'
[ INFO] [1648875306.919740150]: Using planning request adapter 'Fix Start State In Collision'
[ INFO] [1648875306.919757965]: Using planning request adapter 'Fix Start State Path Constraints'
[INFO] [1648875306.966930, 0.000000]: Controller Spawner: Waiting for service controller_manager/load_controller
[ INFO] [1648875307.217354611]: Finished loading Gazebo ROS API Plugin.
[ INFO] [1648875307.221033716]: waitForService: Service [/gazebo/set_physics_properties] has not been advertised, waiting...
[ INFO] [1648875307.251564573]: Finished loading Gazebo ROS API Plugin.
[ INFO] [1648875307.252612655]: waitForService: Service [/gazebo_gui/set_physics_properties] has not been advertised, waiting...
[INFO] [1648875307.325505, 0.000000]: Loading model XML from ros parameter robot_description
[INFO] [1648875307.332912, 0.000000]: Waiting for service /gazebo/spawn_urdf_model
[ INFO] [1648875308.210555281]: Stereo is NOT SUPPORTED
[ INFO] [1648875308.210602521]: OpenGL device: NVIDIA GeForce RTX 2070 SUPER/PCIe/SSE2
[ INFO] [1648875308.210617973]: OpenGl version: 4.6 (GLSL 4.6).
[ INFO] [1648875308.962092974]: waitForService: Service [/gazebo/set_physics_properties] is now available.
[ INFO] [1648875308.983305148, 0.010000000]: Physics dynamic reconfigure ready.
[INFO] [1648875309.141363, 0.160000]: Calling service /gazebo/spawn_urdf_model
[INFO] [1648875309.240772, 0.194000]: Spawn status: SpawnModel: Successfully spawned entity
[ INFO] [1648875309.267238960, 0.194000000]: Loading gazebo_ros_control plugin
[ INFO] [1648875309.267310756, 0.194000000]: Starting gazebo_ros_control plugin in namespace: /crane_x7
[ INFO] [1648875309.267667105, 0.194000000]: gazebo_ros_control plugin is waiting for model URDF in parameter [/robot_description] on the ROS param server.
[urdf_spawner-6] process has finished cleanly
log file: /home/daisuke/.ros/log/0fef90ac-b241-11ec-b3b9-a85e45a7a8b0/urdf_spawner-6*.log
[ INFO] [1648875309.460086509, 0.194000000]: Loaded gazebo_ros_control.
[ INFO] [1648875309.464029896, 0.194000000]: MimicJointPlugin loaded! Joint: "crane_x7_gripper_finger_a_joint", Mimic joint: "crane_x7_gripper_finger_b_joint", Multiplier: 1, Offset: 0, MaxEffort: 4, Sensitiveness: 0
[INFO] [1648875309.640364, 0.370000]: Controller Spawner: Waiting for service controller_manager/switch_controller
[INFO] [1648875309.643008, 0.373000]: Controller Spawner: Waiting for service controller_manager/unload_controller
[INFO] [1648875309.644453, 0.374000]: Loading controller: joint_state_controller
[INFO] [1648875309.649340, 0.379000]: Loading controller: arm_controller
[INFO] [1648875309.674638, 0.404000]: Loading controller: gripper_controller
[INFO] [1648875309.691541, 0.421000]: Controller Spawner: Loaded controllers: joint_state_controller, arm_controller, gripper_controller
[INFO] [1648875309.694397, 0.423000]: Started controllers: joint_state_controller, arm_controller, gripper_controller
[ INFO] [1648875309.965816923, 0.696000000]: Added FollowJointTrajectory controller for /crane_x7/arm_controller
[ INFO] [1648875310.266424908, 0.995000000]: Added GripperCommand controller for /crane_x7/gripper_controller
[ INFO] [1648875310.266508490, 0.995000000]: Returned 2 controllers in list
[ INFO] [1648875310.278192434, 1.007000000]: Trajectory execution is managing controllers
Loading 'move_group/ApplyPlanningSceneService'...
Loading 'move_group/ClearOctomapService'...
Loading 'move_group/MoveGroupCartesianPathService'...
Loading 'move_group/MoveGroupExecuteTrajectoryAction'...
Loading 'move_group/MoveGroupGetPlanningSceneService'...
Loading 'move_group/MoveGroupKinematicsService'...
Loading 'move_group/MoveGroupMoveAction'...
Loading 'move_group/MoveGroupPickPlaceAction'...
Loading 'move_group/MoveGroupPlanService'...
Loading 'move_group/MoveGroupQueryPlannersService'...
Loading 'move_group/MoveGroupStateValidationService'...
[ INFO] [1648875310.342491781, 1.072000000]: 

********************************************************
* MoveGroup using: 
*     - ApplyPlanningSceneService
*     - ClearOctomapService
*     - CartesianPathService
*     - ExecuteTrajectoryAction
*     - GetPlanningSceneService
*     - KinematicsService
*     - MoveAction
*     - PickPlaceAction
*     - MotionPlanService
*     - QueryPlannersService
*     - StateValidationService
********************************************************

[ INFO] [1648875310.342558238, 1.072000000]: MoveGroup context using planning plugin ompl_interface/OMPLPlanner
[ INFO] [1648875310.342718480, 1.072000000]: MoveGroup context initialization complete

You can start planning now!

[ INFO] [1648875312.568723445, 3.291000000]: Loading robot model 'crane_x7'...
[ INFO] [1648875312.568756671, 3.291000000]: No root/virtual joint specified in SRDF. Assuming fixed joint
[ INFO] [1648875312.658531059, 3.381000000]: Starting planning scene monitor
[ INFO] [1648875312.660114652, 3.382000000]: Listening to '/move_group/monitored_planning_scene'
[ INFO] [1648875312.837749499, 3.560000000]: Constructing new MoveGroup connection for group 'arm' in namespace ''
[ INFO] [1648875313.874939382, 4.590000000]: Ready to take commands for planning group arm.
[ INFO] [1648875313.875023957, 4.590000000]: Looking around: no
[ INFO] [1648875313.875060998, 4.590000000]: Replanning: no
[WARN] [1648875337.082548, 27.536000]: Controller Spawner couldn't find the expected controller_manager ROS interface.
[controller_gazebo-3] process has finished cleanly
log file: /home/daisuke/.ros/log/0fef90ac-b241-11ec-b3b9-a85e45a7a8b0/controller_gazebo-3*.log
```

```
$ cat controller_gazebo-3.log
[rospy.client][INFO] 2022-04-02 13:55:06,854: init_node, name[/controller_gazebo], pid[8708]
[xmlrpc][INFO] 2022-04-02 13:55:06,854: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2022-04-02 13:55:06,854: ROS Slave URI: [http://192.168.100.2:35219/]
[xmlrpc][INFO] 2022-04-02 13:55:06,854: Started XML-RPC server [http://192.168.100.2:35219/]
[rospy.impl.masterslave][INFO] 2022-04-02 13:55:06,854: _ready: http://192.168.100.2:35219/
[xmlrpc][INFO] 2022-04-02 13:55:06,855: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2022-04-02 13:55:06,855: Registering with master node http://192.168.100.2:11311
[rospy.init][INFO] 2022-04-02 13:55:06,954: registered with master
[rospy.rosout][INFO] 2022-04-02 13:55:06,955: initializing /rosout core topic
[rospy.rosout][INFO] 2022-04-02 13:55:06,957: connected to core topic /rosout
[rospy.simtime][INFO] 2022-04-02 13:55:06,957: initializing /clock core topic
[rospy.simtime][INFO] 2022-04-02 13:55:06,958: connected to core topic /clock
[rosout][INFO] 2022-04-02 13:55:06,966: Controller Spawner: Waiting for service controller_manager/load_controller
[rospy.internal][INFO] 2022-04-02 13:55:07,337: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2022-04-02 13:55:09,087: topic[/clock] adding connection to [http://192.168.100.2:43279/], count 0
[rosout][WARNING] 2022-04-02 13:55:37,082: Controller Spawner couldn't find the expected controller_manager ROS interface.
[rospy.core][INFO] 2022-04-02 13:55:37,084: signal_shutdown [atexit]
[rospy.internal][INFO] 2022-04-02 13:55:37,086: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2022-04-02 13:55:37,086: topic[/clock] removing connection to http://192.168.100.2:43279/
[rospy.internal][WARNING] 2022-04-02 13:55:37,087: Unknown error initiating TCP/IP socket to 192.168.100.2:37327 (http://192.168.100.2:43279/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2022-04-02 13:55:37,087: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 846, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2022-04-02 13:55:37,088: atexit
```

# その他
<!-- 何かあれば追記してください -->

--- a/crane_x7_control/launch/crane_x7_fake_control.launch
+++ b/crane_x7_control/launch/crane_x7_fake_control.launch
@@ -17,11 +17,4 @@
     <rosparam file="$(find crane_x7_control)/config/crane_x7_gazebo_control.yaml" command="load" />
   </group>
 
-  <node name="controller_gazebo" 
-    pkg="controller_manager" 
-    type="spawner" 
-    respawn="false" 
-    output="screen"
-    args="gazebo_ros_control" />
-
 </launch>

